### PR TITLE
update bpaf to 0.5.0

### PR DESCRIPTION
**Pull request by pacak**
Next version, checking I'm not breaking anything. Shouldn't change the behavior, but I adjusted one of the help messages so it fits better within reasonable screen width.

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -45,8 +45,8 @@ fn opts() -> Opts {
     let image_format = short('f')
         .long("format")
         .help(
-            "image format of generated image. Only PNG or SVG is valid (case-insensitive). \
-            PNG will be used if not specified.",
+            "image format of generated image. Only PNG or SVG is valid (case-insensitive).
+PNG will be used if not specified.",
```
Comment (JohnTitor) [inline src/main.rs:49]: Why is this change needed?

Comment (pacak) [reply src/main.rs:49]: It's not needed but it makes it so the help message fits into 120 columns wide screen rather than 150 one. I can drop it if you prefer.

Comment (JohnTitor) [reply src/main.rs:49]: Makes sense, thanks for clarifying!

**Review by JohnTitor — APPROVED** [review]

File: src/main.rs (hunk 1 above)
Comment (pacak) [reply src/main.rs:49]: While at it - I released 0.5.3, if you just bump the version users will get better output in case of a typo in a flag/command name. No other changes should be required.

Comment (pacak) [reply src/main.rs:49]: Well, regenerate a lock file if you ship one.

Comment (JohnTitor) [reply src/main.rs:49]: Thanks for the tip! Updated it along with other deps: https://github.com/JohnTitor/ranpha/commit/ef2a49b0c2224462a788b5f3a95d9783ca92c3c2
